MODULE.bazel

```diff
@@ -6,10 +6,10 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
+bazel_dep(name = "aspect_bazel_lib", version = "1.42.3")
 
 # Note: only used for provider symbols, we don't spawn nodejs actions
-bazel_dep(name = "aspect_rules_js", version = "1.29.2")
+bazel_dep(name = "aspect_rules_js", version = "1.41.2")
```

Comment:
Somewhere between 1.29.2 and latest release support for Bazel 7 was fixed....

Reply:
FYI https://pro.aspect.build/guides/bzlmod.pro#bazel-7-minimum-versions tells you exactly which version is minimum

Reply:
Ahh. That makes it easier 👍